src/main/java/com/hyperlocal/server/MerchantController.java

```diff
@@ -31,7 +35,7 @@ public MerchantController() {
    * @param postInputString JSON serialized {@link Merchant} details
    * @return CompletableFuture of the updated Merchant
    */
-  @PutMapping("/v1/merchants/{merchantID}")
+  // @PutMapping("/v1/merchants/{merchantID}")
```

Comment:
Why is this commented out?

Reply:
I had made the API to create merchant idempotent in order to reduce the number of network calls made by the frontend. This makes API to update merchant irrelevant in the context of this microapp.

Reply:
Anyway, thanks for pointing this out - it's a better practice to remove the unused method rather than to comment its route out. I have made that change.

Reply:
It is unclear how create is made idempotent, It still performs INSERT right?

Reply:
Yes - the SQL query now checks if a merchant with the given unique ID already exists. 
https://github.com/googleinterns/hyperlocal-gpay-spot/blob/73e27f848b73bed64d13f578ff246d82b28b8955/src/main/java/com/hyperlocal/server/Constants.java#L38